includes/class-wc-payments-features.php

```diff
@@ -13,8 +13,8 @@
  * WC Payments Features class
  */
 class WC_Payments_Features {
-	const UPE_FLAG_NAME = '_wcpay_feature_upe';
-
+	const PLATFORM_CHECKOUT_FLAG_NAME   = 'platform_checkout_eligible';
```

Comment:
What do think about changing the `PLATFORM_CHECKOUT_FLAG_NAME` to `PLATFORM_CHECKOUT_ELIGIBLE_FLAG_NAME` ? 	The name used could be misleading

Reply:
Makes sense to me, updated in 704ab29f5533d1673e631ad089ed54f9276e018d.